docs-client/src/containers/MethodPage/index.tsx

```diff
@@ -212,6 +217,15 @@ export default class MethodPage extends React.PureComponent<Props, State> {
                 <Typography variant="body1" paragraph />
                 {this.state.additionalHeadersOpen && (
                   <>
+                    <>
+                      {this.state.exampleHeaders.length > 0 && (
+                        <Dropdown
+                          placeholder="Select an example headers"
```

Comment:
`Select an example headers...` ? (... to indicate that there's something to select after clicking on it?)

Reply:
Updated. Thanks~!